packages/tradershub/src/features/cfd/modals/MT5PasswordModal/PasswordComponent.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useAccountStatus } from '@deriv/api';
+import { Provider } from '@deriv/library';
+import { SentEmailContent } from '../../../../components';
+import useMT5AccountHandler from '../../../../hooks/useMT5AccountHandler';
+import MT5PasswordIcon from '../../../../public/images/ic-mt5-password.svg';
+import { PlatformDetails } from '../../constants';
+import { CreatePassword, EnterPassword } from '../../screens';
+
+type TPasswordComponentProps = {
+    password: string;
+    setPassword: (password: string) => void;
+};
+
+const PasswordComponent = ({ password, setPassword }: TPasswordComponentProps) => {
+    const { data: accountStatus } = useAccountStatus();
+    const { show } = Provider.useModal();
+    const { getCFDState } = Provider.useCFDContext();
+
+    const marketType = getCFDState('marketType') ?? 'all';
+    const platform = getCFDState('platform') ?? 'mt5';
+    const selectedJurisdiction = getCFDState('selectedJurisdiction') ?? 'maltainvest';
+
+    const isMT5PasswordNotSet = accountStatus?.is_mt5_password_not_set;
+    const { createMT5AccountLoading, handleSubmit, isCreateMT5AccountError, tradingPlatformPasswordChangeLoading } =
+        useMT5AccountHandler({
+            marketType,
+            selectedJurisdiction,
+        });
+
+    if (isMT5PasswordNotSet) {
+        return (
+            <CreatePassword
+                icon={<MT5PasswordIcon />}
+                isLoading={tradingPlatformPasswordChangeLoading || createMT5AccountLoading}
```

Comment:
In my suggestion if a value is repeated twice then it should be made a variable for better readability .
Any thoughts @shahzaib-deriv @matin-deriv @thisyahlen-deriv

Reply:
can can anything for better readability